test/fixtures/v5/fork/straight_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Mantengase recto en el cruce",
         "fr": "Rester tout droit à l'embranchement",
         "id": "Tetap lurus di pertigaan",
+        "it": "Mantieni la Continua dritto al bivio",
```

Comment:
This is also awkward in English: “Keep straight at the fork”. However, I think it’s unlikely that the Directions API would ever output this instruction. I think the API would omit the instruction entirely, unless the destination street has a different name, in which there would be a `continue` instruction. @bsudekum, is that correct?